server/index.js

```diff
@@ -28,6 +28,6 @@ http.createServer(app).listen(serverPort, function () {
 });
 
 //setting the path to the database 
-global.databasePath = './database/load.json'
```

Comment:
kindly update the port of TAR to the one which is mapped for 2.1.0 . 
Also, please update in dockerFile as per documentation.

Reply:
Done